guide/src/ref/packet-log.md

```diff
@@ -103,18 +230,164 @@ protocol "tcp.flow_id:32,tcp.seq:32,tcp.ack:32,ip.id:16,ip.frag:16,ip.checksum:1
+|                            Reserved                           |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                                                               |
++                                                               +
+|                                                               |
++                                                               +
+|                                                               |
++                                                               +
+|                                                               |
++                                                               +
+|                                                               |
++                     Options on SYN/SYN_ACK                    +
+|                                                               |
++                                                               +
+|                                                               |
++                                                               +
+|                                                               |
++                                                               +
+|                                                               |
++                                                               +
+|                                                               |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 ```
 
-Where:
+```bash
+protocol "LH.length:12,LH.ty.:4,FLE.length:12,FLE.ty.:4,Flow Id:32,src.ip:32,dst.ip:32,src.port:16,dst.port:16,Base Timestamp:64,Reserved:32,Options on SYN/SYN_ACK:320"
+```
+
+where `FLE` is short for FLow Entry, and the meaning of each field is as follows:
+
+- `FLE.length` is the length for the whole flow entry. `FLE.type` could be 0 only for now, denoting a TCP/IPv4 flow.
+- `Flow id`: The identifier of the flow. The high 16 bits are the interface ID where the packet enters Rattan, and the low 16 bits are the incremental flow counter.
+- `Base Timestamp`: The base timestamp in nanoseconds (this unit is different from the `GP.timestamp` field) for this flow.
+- `src.ip`, `dst.ip`, `src.port`, `dst.port`: The source IPv4 address, destination IPv4 address, source TCP port number, description TCP port number.
+- `Options on SYN/SYN_ACK`: As flows are uni-directional here, only one valid of SYN(TCP client side) or SYN_ACK (TCP server side) would exist in a TCP connection. The TCP options fileds in such pakcets are recorded in raw here. This field has a fixed length of 40 Bytes, and zero-padding at the end if the tcp options as less than 40 Bytes. 40 Bytes is the upper limit in TCP protocol: a TCP header can be at most 15 \* 4 = 60 Bytes in length, and the first 20 bytes are used for fixed fields in the TCP header.
+
+#### Chunk Prologue Entry
+
+A chunk prologue entry is a log entry with `LH.typ` = 15, which is used in raw-mode packet logs.
```

Comment:
What is `LH.typ`? We only introduce `LH.ty.` in our spec and the the following block you are showing only have `LH.ty.`